system_setup/ui/views/wslconfbase.py

```diff
@@ -95,9 +95,11 @@ class WSLConfigurationBaseView(BaseView):
     excerpt = _(
         "In this page, you can configure Ubuntu WSL options to your needs.\n")
 
-    def __init__(self, controller, configuration_data):
+    def __init__(self, controller, configuration_data, variant):
         self.controller = controller
-
+        self.is_conf = variant == "wsl_configuration"
+        if self.is_conf:
+            self.title = _("WSL configuration - Base options")
```

Comment:
We should always use that title without using the variant.

Reply:
This one is for In wsl_configuration mode the UI shows ”base”/”advanced” screens though, People would be confused that there are two configuration pages with the same title, otherwise we should change the description

Reply:
Yes. but my suggestion was to always use that title rather. Those are the base options and can show them as such in setup mode.

Reply:
Addressed